components/Footer.tsx

```diff
@@ -1,53 +1,96 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
 export default function Footer() {
-  const footerNavigation = [
+  const navigation = [
+    { name: 'Home', href: '/' },
+    // { name: 'Projects', href: '/projects' },
+    { name: 'Supporters', href: '/supporters' },
+    // { name: 'Contact', href: '/contact' },
```

Comment:
You can uncomment contact and put the same link as we have on the header

Reply:
Done